### PR TITLE
librdkafka: update to 1.8.2.

### PR DESCRIPTION
**Pull request by knusbaum**
#### Testing the changes
- I tested the changes in this PR: **YES**

### CODE REVIEW

**Comment by ericonr** [comment]
How did you test?

Just want to make sure `kafkacat` works fine with it.

**Comment by knusbaum** [comment]
@ericonr

I just rebuilt and tested kafkacat with a live kafka broker, producer, and consumer.
I am not able to exhaustively test kafkacat, but it compiles and basic commands (monitoring a topic) work.

File: srcpkgs/kafkacat/template [hunk 1]
```diff
@@ -1,7 +1,8 @@
 # Template file for 'kafkacat'
 pkgname=kafkacat
 version=1.6.0
-revision=1
+revision=2
```
Comment (knusbaum) [inline srcpkgs/kafkacat/template:4]: Looks like this package changed out from under us. (different checksum, different `wrksrc`), but it *does* bulid and run. 

Not sure how we handle situations like this.

**Comment by knusbaum** [comment]
Rebased.

**Pull request merged**